templates/web-deployment.yaml

```diff
@@ -126,6 +126,9 @@ spec:
       {{- if $.Values.web.extraVolumeMounts }}
         {{- toYaml $.Values.web.extraVolumeMounts | nindent 8 }}
       {{- end }}
+        {{- with $.Values.web.containerExtraSpec }}
+          {{- toYaml . | nindent 10 }}
+        {{- end }}
```

Comment:
```suggestion
      {{- with $.Values.web.containerExtraSpec }}
        {{- toYaml . | nindent 8 }}
      {{- end }}
```

Reply:
Resolved in error? If not can you say here why this has a different indentation to the `volumeMounts` block above?